### ISSUE TITLE
Fix scoring concept

### ISSUE THREAD

**Issue by bjbestnic** [nic-dgl213-f21/dgl213-module1-ItsJustJarvis#1]
To win, with one move to go, clicking on the *wrong* color enough times will produce a high enough score to win.  Otherwise, it's not clear how to win without resorting to this trick.

**Comment by ItsJustJarvis** [comment]
I am slightly confused by your statement, "Otherwise, it's not clear how to win without resorting to this trick.". 

My intention was for the player to not always win this game, if they make small moves rather than big flood fills. However, I do see what you are saying with the selection of wrong colours leading to an eventual win.  I considered this when addressing the issue, but I don't think my solution has eliminated the problem you are referring to.

I feel like part of this issue was generally focussed around player understanding, and proper presentation of information. I have simplified things a little bit in an attempt to address this. Rather than waiting till the end of the game to calculate the final score, the player score is consistently updated based on number of clicks taken. This way the player can see how close they are to surpassing the 200 point limit, or if they are already in a winning position. Additionally, it will provide more feedback to the player showing their lack of point gains when making moves that are too small. 

I considered changing the system to award points based on how many adjacent cells were gained, or added to the filled area (rather than the changed cells) but I struggled to find a way to make this work (it was producing more bugs).
